src/main/java/com/alexnederlof/jasperreport/CompileTask.java

```diff
@@ -55,24 +55,13 @@ public CompileTask(File source, File destination, Log log, boolean verbose) {
      */
     @Override
     public Void call() throws Exception {
-        OutputStream out = null;
-        InputStream in = null;
-        try {
-            out = new FileOutputStream(destination);
-            in = new FileInputStream(source);
+        try (OutputStream out = new FileOutputStream(destination); InputStream in = new FileInputStream(source)) {
```

Comment:
LGTM

Reply:
I had to look that up, I'm new to gitHub. If that means Looks Good To Me: thanks!